mubench.pipeline/tasks/implementations/publish_findings.py

```diff
@@ -25,6 +26,7 @@ def __init__(self, experiment_id: str, compiles_base_path: str, review_site_url:
                  review_site_password: str = ""):
         super().__init__()
         self.max_files_per_post = 20  # 20 is PHP's default limit to the number of files per request
+        self.max_file_size_per_post = 5000  # choose a moderate value since we only approximate upload size
```

Comment:
`max_post_size_in_bytes`?

PHP defaults to 2MB per file and 8MB for the whole request. I would probably set 7MB here, to be on the safe side.

Looking at this, we should consider the size of the image files as well... can we do that?